### PR TITLE
Improve tasks logging. Send pricelist to media

### PR DESCRIPTION
**Pull request by zzzevaka**

### CODE REVIEW

File: connect_ext_ppr/services/pricing.py [hunk 1]
```diff
@@ -56,24 +56,63 @@ def apply_pricelist_to_marketplace(
...
+        deployment_request.id,
+        marketplace.pricelist_id,
+        marketplace.marketplace,
+        reseller_id,
+    )
     excel_file, file_name, dataset = _prepare_file(
         client=connect_client,
         batch_id=marketplace.pricelist_id,
     )
+    create_dr_file_to_media(
```
Comment (Hairash) [inline connect_ext_ppr/services/pricing.py:84]: Is it just a backup file?
I suggest to write the filename construction principle in the DR article like we did for "Marketplace update" and "Delegate to L2" tasks.

**Review by Hairash — APPROVED** [review]
I propose to make one commit instead of 3 following the agreement we have for the PRs if you don't mind

**Comment by zzzevaka** [comment]
> I propose to make one commit instead of 3 following the agreement we have for the PRs if you don't mind

Could you refer to which agreement you mean?

There are 3 changes unrelated. I would prefer to leave it as is.

File: connect_ext_ppr/services/pricing.py (hunk 1 above)
Comment (zzzevaka) [reply connect_ext_ppr/services/pricing.py:85]: fixed

**Comment by Hairash** [comment]
> > I propose to make one commit instead of 3 following the agreement we have for the PRs if you don't mind
> 
> Could you refer to which agreement you mean?
> 
> There are 3 changes that are not related to each other. I would prefer to leave it as is.

Maybe it was agreement inside our team. I found only this:
https://confluence.int.zone/display/CONNECT/GIT-based+development+process#GITbaseddevelopmentprocess-P2.MakesurethatallcommitsinyourPRisrelevantformodulehistory
Anyway it's not critical at all. I just found this line and was surprised
https://github.com/cloudblue/connect-extension-xvs/pull/119/commits/fa544c2d7347f79a617d1bca32614c30141036da#diff-15c6957d32ad5a0f63fc907f8aa65253c0f18e8cae078bb8369538ad2a54beadR213
But I'm ok to leave it as is.

**Pull request merged**